packages/test/src/test-otel.ts

```diff
@@ -604,3 +679,43 @@ test('Can replay signal workflow from 1.13.1', async (t) => {
     );
   });
 });
+
+test('Can replay smorgasbord from 1.13.2', async (t) => {
+  const hist = await loadHistory('otel_smorgasbord_1_13_2.json');
+  await t.notThrowsAsync(async () => {
+    await Worker.runReplayHistory(
+      {
+        workflowBundle: await createTestWorkflowBundle({
+          workflowsPath: require.resolve('./workflows'),
+          workflowInterceptorModules: [require.resolve('./workflows/otel-interceptors')],
+        }),
+        interceptors: {
+          workflowModules: [require.resolve('./workflows/otel-interceptors')],
+          activity: [
+            (ctx) => ({
+              inbound: new OpenTelemetryActivityInboundInterceptor(ctx),
+              outbound: new OpenTelemetryActivityOutboundInterceptor(ctx),
+            }),
+          ],
+        },
+      },
+      hist
+    );
+  });
+});
+
+// Skipped as we only care that it compiles
+test.skip('otel interceptors are complete', async (t) => {
```

Comment:
I build and checked at the docs and we end up leaking the `Required` type. Switched to a compile time check that these types satisfy the narrower type, but we only expose implementing the wider interface to users.

Reply:
Good thinking.